src/mlpack/core/dists/discrete_distribution.hpp

```diff
@@ -168,6 +168,7 @@ class DiscreteDistribution
     return log(Probability(observation));
   }
 
+  void LogProbability(const arma::mat& x, arma::vec& logProbabilities) const;
```

Comment:
Can you add the comment in your implementation to here too?  I think it's more likely developers might look here for documentation. :+1: